From fa07c1a30be7144f5d3292208f5d58799c92189f Mon Sep 17 00:00:00 2001
From: Meng Yu <yumeng18@huawei.com>
Date: Tue, 6 Apr 2021 10:32:59 +0800
Subject: [PATCH] crypto: ecc - delete a useless function declaration
Git-commit: fa07c1a30be7144f5d3292208f5d58799c92189f
Patch-mainline: v5.13-rc1
References: bsn#12


This function declaration has been added in 'ecc_curve.h',
delete it in 'crypto/ecc.h'.

Fixes: 4e6602916bc6(crypto: ecdsa - Add support for ECDSA ...)
Signed-off-by: Meng Yu <yumeng18@huawei.com>
Signed-off-by: Herbert Xu <herbert@gondor.apana.org.au>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
Signed-off-by: Geliang Tang <geliang.tang@suse.com>
---
 crypto/ecc.h | 8 --------
 1 file changed, 8 deletions(-)

--- a/crypto/ecc.h
+++ b/crypto/ecc.h
@@ -97,14 +97,6 @@ static inline void ecc_swap_digits(const
 }
 
 /**
- * ecc_get_curve()  - Get a curve given its curve_id
- * @curve_id:  Id of the curve
- *
- * Returns pointer to the curve data, NULL if curve is not available
- */
-const struct ecc_curve *ecc_get_curve(unsigned int curve_id);
-
-/**
  * crypto_ecdh_shared_secret() - Compute a shared secret
  *
  * @curve_id:		id representing the curve to use
